configuration/configurator/schemas/SchemaAttributes.cpp

```diff
@@ -195,15 +195,17 @@ void CAttribute::populateEnvXPath(StringBuffer strXPath, unsigned int index)
 {
     assert(this->getName() != nullptr);
 
-    const char *pChar = strrchr(strXPath.str(),'[');
-    assert(pChar != nullptr && strlen(pChar) >= 3);
+
```

Comment:
No, I didn't.  This change is from I run debug mode (assert enabled).
This is Gleb's new code. I think he didn't test for all xsd files. Some XPath strings do not have the pattern he assumed, i.e include '['. 
But actually later I found lots of asserts will throw exceptions.  So it cannot run in debug mode. 
I will remove this change.